src/components/Icons/index.js

```diff
@@ -79,3 +79,10 @@ export { default as User } from './User';
 export { default as Webhooks } from './Webhooks';
 export { default as XmarkLarge } from './XmarkLarge';
 export { default as PuzzlePiece } from './PuzzlePiece';
+export { default as AppWindows } from './AppWindows';
```

Comment:
Small ask: can you export them in alphabetical order? So they show this way in the story. At first glance I thought you weren't exporting them.

![image](https://github.com/lob/ui-components/assets/83967528/587af2da-4da8-44c1-b6c1-f3ffba9a7366)

Reply:
Nit: is it possible the new icons look smaller than the rest? Or is it just me? I could be wrong.

Reply:
> Nit: is it possible the new icons look smaller than the rest? Or is it just me? I could be wrong.e72e
Yeah, agreed. These were the relative sizes from the figma, so I went 1:1, but imo we should probably standardize how these get sized. I'm gonna leave them as-is for now, but let's circle back to this when we discuss the design system and common components.